src/Confluent.Kafka/WatermarkOffsets.cs

```diff
@@ -33,7 +33,7 @@ public class WatermarkOffsets
         ///     The offset of the earlist message in the topic/partition.
         /// </param>
         /// <param name="high">
-        ///     The offset of the last committed message in the topic/partition.
+        ///     The offset of the last stored message in the topic/partition.
```

Comment:
probably should give your jet lag a break and go to bed! 

the initial design docs for this actually use the word committed in relation to high watermark.